aether-kernel/aether/kernel/api/models.py

```diff
@@ -279,6 +315,8 @@ class Entity(ExportModelOperationsMixin('kernel_entity'), models.Model):
 
     projectschema = models.ForeignKey(to=ProjectSchema, on_delete=models.SET_NULL, null=True)
     submission = models.ForeignKey(to=Submission, on_delete=models.SET_NULL, blank=True, null=True)
+    mapping = models.ForeignKey(to=Mapping, on_delete=models.SET_NULL, null=True)
```

Comment:
With this new field maybe you can include it the project check in the save method too :thinking:

Reply:
Mappings are no longer directly linked to project. not sure the check is necessary.

Reply:
Yes, they still are (with that redundant field) :wink: But it's not necessary to include it the check...